manuscript/better-environment.md

```diff
@@ -1,7 +1,8 @@
 {#the-environment}
 # A good work environment
+If you already know how to set up the perfect development environment for modern JavaScript work, go ahead and skip this section. If you don't, there are a few things you should know.
```

Comment:
Nice addition. I was like "Wait, I don't remember writing that, but it sounds so right"